docs/content/latest/benchmark/tpcc-ysql/100-wh/tpcc-ysql.md

```diff
@@ -28,14 +53,15 @@ $ ./tpccbenchmark --create=true --load=true --execute=true \
   --warehouses=100
 ```
 
-### TPC-C Benchmark Results
+## 4. TPC-C Benchmark Results
 
 Once the execution is done the TPM-C number along with the efficiency is printed.
 
 ```
-01:25:34,669 (DBWorkload.java:880) INFO  - Rate limited reqs/s: Results(nanoSeconds=1800000482491, measuredRequests=423634) = 235.35215913594521 requests/sec
-01:25:34,669 (DBWorkload.java:885) INFO  - Num New Order transactions : 186826, time seconds: 1800
-01:25:34,669 (DBWorkload.java:886) INFO  - TPM-C: 6227
-01:25:34,669 (DBWorkload.java:887) INFO  - Efficiency : 96.84292379471229
-01:25:34,671 (DBWorkload.java:722) INFO  - Output Raw data into file: results/oltpbench.csv
+18:01:57,149 (DBWorkload.java:894) INFO  - ======================================================================
+18:01:57,149 (DBWorkload.java:895) INFO  - Rate limited reqs/s: Results(nanoSeconds=1800000367051, measuredRequests=85376) = 47.43110143909266 requests/sec
```

Comment:
Separately, we should remove the term "Rate limited" and instead just call it "Throughput".
Along similar lines, we should enhance the output to provide latency numbers too and also provide a split of throughput and latency across the 5 workload types.

Reply:
Right that makes sense.